presto-spark-base/src/main/java/com/facebook/presto/spark/execution/PrestoSparkAdaptiveQueryExecution.java

```diff
@@ -287,7 +295,7 @@ public Void apply(Try<MapOutputStatistics> result)
             executedFragments.add(successEvent.getFragmentId());
             Optional<PlanNodeStatsEstimate> runtimeStats = createRuntimeStats(successEvent.getMapOutputStats());
             // Re-optimizations here.
-
+            // Log remaining plan before and after optimization
```

Comment:
I think this will will be too much logging.  We do already log if an optimization was triggered, though we won't know at what point in the plan.

Reply:
can you remove this comment?